<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="enabledB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Włącz wtyczkę</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPushButton" name="addB">
     <property name="text">
      <string>Dodaj generator tonów do playlisty</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Ilość kanałów:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="monoB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Mono</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="stereoB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Stereo</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="fourB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>4 kanały</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="sixB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>6 kanałów</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="eightB">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>8 kanałów</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Ton o częstotliwości:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Kanał 1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="hzB">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Kanał 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QSpinBox" name="hzB_2">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Kanał 3:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QSpinBox" name="hzB_3">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Kanał 4:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QSpinBox" name="hzB_4">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Kanał 5:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="hzB_5">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Kanał 6:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="hzB_6">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Kanał 7:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Kanał 8:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="hzB_7">
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="hzB_8">
        <property name="maximum">
         <number>96000</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Hz</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="ApplyB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Zastosuj</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="srateB">
     <property name="minimum">
      <number>4000</number>
     </property>
     <property name="maximum">
      <number>192000</number>
     </property>
     <property name="value">
      <number>44100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Częstotliwość próbkowania</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>addB</tabstop>
  <tabstop>srateB</tabstop>
  <tabstop>monoB</tabstop>
  <tabstop>stereoB</tabstop>
  <tabstop>fourB</tabstop>
  <tabstop>sixB</tabstop>
  <tabstop>eightB</tabstop>
  <tabstop>hzB</tabstop>
  <tabstop>hzB_2</tabstop>
  <tabstop>hzB_3</tabstop>
  <tabstop>hzB_4</tabstop>
  <tabstop>hzB_5</tabstop>
  <tabstop>hzB_6</tabstop>
  <tabstop>hzB_7</tabstop>
  <tabstop>hzB_8</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
